addons/escoria-core/plugin.gd

```diff
@@ -92,6 +91,14 @@ func set_escoria_ui_settings():
 
 # Prepare the settings in the Escoria main category
 func set_escoria_main_settings():
+	if !ProjectSettings.has_setting("escoria/main/game_version"):
```

Comment:
In fact, I just realized that the game version is not important here. Since Escoria totally manages savegames (no user-defined data to save), it is up to Escoria to manage changes in the way savegames would differ between Escoria versions.

So, this will have to be changed in favor of Escoria version instead.

Reply:
We talked on Discord about this. Maybe we should add a game version as well, because if globals change in a game, this could be fixed in some kind of hook by the developer. Maybe just put it in the Todo.